lib/connections/snowflake.js

```diff
@@ -36,8 +37,10 @@ connection.prototype.connect = function(callback){
     self.connection = snowflake.createConnection(self.options);
 
     self.connection.connect(function(err, conn) {
+        self.connection_id = conn.getId();
```

Comment:
we should first check for err, if not error then we should do this.. i think its a wrong place to put this line. Also log the connection id as info when the connection is successful.